backend/social_tasks/sorsa_client.py

```
@@ -119,6 +120,57 @@ def is_following(self, actor_handle: str, target_handle: str) -> tuple[bool, dic
+        except ValueError as exc:
+            raise SorsaError('Sorsa returned non-JSON response') from exc
+
+        if response.status_code >= 400:
+            raise SorsaError(f'Sorsa error {response.status_code}: {data}')
+
+        full_text = data.get('full_text')
+        if not isinstance(full_text, str):
+            raise SorsaError('Unexpected tweet-info response (no full_text)')
+        user_payload = data.get('user')
+        if not isinstance(user_payload, dict):
+            raise SorsaError('Unexpected tweet-info response (no user object)')
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)